repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -155,6 +156,34 @@ export const getServicesByNocCodeAndDataSource = async (nocCode: string, source:
     }
 };
 
+export const checkBodshasServicesByNoc = async (nationalOperatorCode: string): Promise<boolean> => {
```

Comment:
I think we need to rename this, because we arent checking if BODS has services, we're checking if the operator has any BODS services in our database. Potentially rename to something like `operatorHasBodsServices` because it returns a boolean.